### DISCUSSION TITLE
Sample app to import

### DISCUSSION THREAD

**Discussion by sz7i** [Budibase/budibase#2311] [Help] [answered]
I'm fully new to budibase and currently just go through the docs. I realized that it would be so much helpful to be able to import a sample app demostrating all the great features implemented in budibase. There are many interesting screenshots and video demos, but it would be really great to be able to get hands-on experience with them.

Anyway awesome project!

**Comment by joebudi** [comment, marked as answer]
Hey!

We are currently working on a couple of templates at the moment. What type of template would you like to see?

**Reply by sz7i** [reply]
The template app topic is not that important for me. I can imagine a sample CRM app can work well. I would mostly appreciate an app which would demostrate various UI design features that you provide.

**Reply by joebudi** [reply]
Understandable. We'll have templates available within the next two months.

**Reply by sz7i** [reply]
Great! Thank you!

**Reply by sz7i** [reply]
Just played with the new templates - excellent! thanks again!
